### DISCUSSION TITLE
hashicorp-consul-rce

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei-templates#6357] [Q&A] [answered]
hi I show my some critical Vulnerability .But how to exploit it.I try but this not works.This can be false positive.
This exploit not allow my for remote command .Any hint.?

github.com/owalid/consul-rce/blob/main/README.md

**Comment by olearycrew** [comment, marked as answer]
@kolor33 I'm not sure if we can help with a POC for this - it very much depends on the target.

You could also join our Discord and see if someone else has encountered this particular vulnerability and has any hints: https://nux.gg/discord.
